src/view/screens/Settings.tsx

```diff
@@ -282,6 +283,34 @@ export const SettingsScreen = withAuthRequired(
             </Text>
           </TouchableOpacity>
 
+          <View style={styles.spacer20} />
+          <Text type="xl-bold" style={[pal.text, styles.heading]}>
+            Appearance
+          </Text>
+          <View>
+            <View style={[styles.linkCard, pal.view, styles.selectableBtns]}>
```

Comment:
Here is an opportunity for refactoring and preventing duplicate codes. You can pull out SelectableBtn from `src/view/com/modals/ContentFilteringSettings.tsx` and create a new one component that reused here and there.

You can use my code from [this](https://github.com/snhasani/social-app/commit/7fdf63869b19205b3fa41590a6525491d19709b0) and [this](https://github.com/snhasani/social-app/commit/a818d13f6231485979696bc8f90900490abf7fcb) commit. It's already there.